SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerPossessionTest.cpp

```diff
@@ -53,16 +52,19 @@ void ACrossServerPossessionTest::PrepareTest()
 				{
 					AssertTrue(PlayerController->HasAuthority(), TEXT("PlayerController should HasAuthority"), PlayerController);
```

Comment:
This assert seems redundant (it will always pass because `PlayerController->HasAuthority()` is checked in the `if` above)

Reply:
Yeah weird, I would suggest removing PlayerController->HasAuthority() check in the if, but its difficult to guess if this was the original intention.

Reply:
I think the one in the if is right, because we want this logic to only execute on the server with auth over the controller.

Reply:
Yup the assert is indeed redundant. It was in the original test which I wasn't comfortable changing when I was working on this file 😄